TMG.EMME/TMG.EMME.Test/TMG.EMME.Test.csproj

```diff
@@ -1,10 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>    
-    <TargetFramework>netcoreapp3.0</TargetFramework>    
+    <TargetFramework>netcoreapp3.1</TargetFramework>    
     <RootNamespace>TMG.Emme.Test</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\calendar.txt" />
+    <Content Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\frequencies.txt" />
+    <Content Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\routes.txt" />
+    <Content Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\stops.txt" />
+    <Content Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\stop_times.txt" />
+    <Content Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\trips.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\routes.csv" />
+    <None Include="bin\Debug\netcoreapp3.1\FrabtiztownGTFS\stop_to_node.csv" />
+  </ItemGroup>
```

Comment:
These files, assuming that they are small, should be included in the test project instead of its output directory.